test/e2e/performanceprofile/functests/utils/nodes/nodes.go

```diff
@@ -434,7 +434,7 @@ func GetCpuSiblings(numaCoreSiblings map[int]map[int][]int, coreKey int) []strin
 	for key := range numaCoreSiblings {
 		for _, c := range numaCoreSiblings[key][coreKey] {
 			cpuSiblings = append(cpuSiblings, strconv.Itoa(c))
-			delete(numaCoreSiblings[key], c)
+			delete(numaCoreSiblings[key], coreKey)
```

Comment:
Uh? I can't quite understand this change. You are deleting the very item you iterate over.

Reply:
The numacoresiblings map is of [numa-Node][core][siblings] 

The reason is  we are using the same map to get reserved , isolated and offline cpus . So once we use cpus for lets say reserved, we delete the core id's from which we picked the siblings so that when we use the map for lets say offline cpus, we want to iterate with what is left so that reserved, offline don't end having same cpu siblings.